esmvaltool/diag_scripts/shared/meta_data.ncl

```diff
@@ -0,0 +1,107 @@
+; #############################################################################
+;  GENERAL ROUTINES FOR WRITING META DATA
+; #############################################################################
+;  Please consider using of extending existing routines before adding new ones.
+;  Check the header of each routine for documentation.
+;
+;  Contents:
+;     procedure ESMValMD
+;
+; #############################################################################
+
+load "interface_scripts/logging.ncl"
+load "interface_scripts/interface.ncl"
```

Comment:
```suggestion
```

Reply:
This solves the above problem.
`interface.ncl` was loaded twice: in `diag_scripts/perfmetrics/main.ncl` and in `diag_scripts/shared/meta_data.ncl`.

Reply:
> This solves the above problem.
> `interface.ncl` was loaded twice: in `diag_scripts/perfmetrics/main.ncl` and in `diag_scripts/shared/meta_data.ncl`.

I had the same problem for several other diagnostics (was really annoying to find that...). Would adding

```ncl
delete([/dataset_info, input_file_info/])
```

at the beginning of the `*_info.ncl` solve this?

Reply:
> I had the same problem for several other diagnostics (was really annoying to find that...). Would adding
>```
>delete([/dataset_info, input_file_info/])
>```
>at the beginning of the *_info.ncl solve this?

No, that would break the ability to read multiple input variables. A good solution would be to check if the interface_scripts/interface.ncl file is already loaded at the top of the interface_scripts/interface.ncl file and either error or simply not execute the remainder of the file if that is the case. I once created an issue for exactly this problem https://github.com/ESMValGroup/ESMValTool/issues/243, but @mattiarighi did not think it was worth the effort to implement.

Reply:
> Alternatively, you can move the ESMValMD procedure to interface.ncl as suggested above.

@mattiarighi or @LisaBock This is fine by me, but can one of you take care of the NCL code?

Reply:
> I once created an issue for exactly this problem #243, but @mattiarighi did not think it was worth the effort to implement.

Loading NCL libraries multiple times is not an issue, indeed.

The problem arises now since `dataset_info` and `input_file_info` are lists, and multiple loading results in list concatenation.

Reply:
I've changed interface.ncl so it only loads the input data description once.